### ISSUE TITLE
Add a logo

### ISSUE THREAD

**Issue by pdesaulniers** [pdesaulniers/wolf-shaper#78]

**Comment by unfa** [comment]
Are graphic design contributions welcome?

**Comment by pdesaulniers** [comment]
Absolutely :)

**Comment by unfa** [comment]
Do you want the "Wolf" thing to become a brand of your products?

**Comment by pdesaulniers** [comment]
Yep, I think Wolf should be a brand, and it should be reflected in the logo. Unless you have a better idea, of course :)

**Comment by unfa** [comment]
I've made a logo. Here's the full history of Inkscape projects:

[Wolf Logo.zip](https://github.com/pdesaulniers/wolf-shaper/files/2385573/Wolf.Logo.zip)

The final design is in "Wolf clean 01.svg"

It looks like this:

![wolf clean 01 svg](https://user-images.githubusercontent.com/491704/45584937-56994000-b8dc-11e8-964e-a27ac5bc7fde.png)
